package.json

```diff
@@ -62,15 +62,15 @@
     "@vitejs/plugin-vue": "^5.0.3",
     "glob": "^8.0.3",
     "install": "^0.13.0",
-    "lodash": "*",
+    "jsdom": "^24.0.0",
+    "lodash": "^4.17.21",
     "npm": "^8.19.2",
     "rollup": "^2.79.1",
     "rollup-plugin-scss": "^4.0.0",
     "rollup-plugin-vue": "^6.0.0",
     "sass": "^1.70.0",
     "scss": "^0.2.4",
     "vite": "^5.0.11",
-    "vitest": "^1.3.0",
-    "jsdom": "^24.0.0"
-   }
+    "vitest": "^1.3.0"
+  }
```

Comment:
please do not commit code which you have not committed, for eg. there is not change to `"vitest": "^1.3.0",`

Reply:
change in line number of vitest due to addition of lodash, reverted back to original.